### DISCUSSION TITLE
Bug in creating new row by trigger automation

### DISCUSSION THREAD

**Discussion by ducnhoxlatui** [Budibase/budibase#11276] [Help] [answered]
Hi guys!
I found the bug in Budibase while trying to run a trigger automation by clicking button and nothing happened. Beside that, testing is ok and still generate new row but nothing happen when click a button. 
![image](https://github.com/Budibase/budibase/assets/71976420/40c22cb7-b26c-4ec8-909a-6d3f59f79a1e)

Thanks for your supporting

**Comment by ConorWebb96** [comment]
Hey @ducnhoxlatui,

Is the app published were you are trying to use the automation?
Can you send through a screenshot of your button actions where you are running this automation? 
Could you provide an app export of the app were the automation is working and I'll have a more in-depth look.

Thanks ahead of time.

**Comment by ducnhoxlatui** [comment]
Hi @ConorWebb96 ,
Yes, i use the published app.
this is the design:
![image](https://github.com/Budibase/budibase/assets/71976420/7833a8be-f86b-4171-a289-615dd13550ba)

Button action:
![image](https://github.com/Budibase/budibase/assets/71976420/2270cfac-f177-430e-a155-ecdb6541aaba)

Automation:
![image](https://github.com/Budibase/budibase/assets/71976420/7b9a2cf6-0945-4c07-940e-404468ae944d)

This is app export:
[My app-export-1689836673767.tar.gz](https://github.com/Budibase/budibase/files/12102994/My.app-export-1689836673767.tar.gz)

Thanks for your supporting!

**Reply by ConorWebb96** [reply, marked as answer]
Hey @ducnhoxlatui,

Thank you for the app export. I've had an in-depth look a the app and seem to have found the issue. It seems for the `App action` trigger automation you need to include a field within the trigger itself, and also have a bindable value passed back from the frontend were you have attached the automation to the button click.

**Trigger screenshot**
<img width="519" alt="Screenshot 2023-07-20 at 08 45 03" src="https://github.com/Budibase/budibase/assets/126772285/2e5e967a-504e-42aa-a594-3c7c26163531">

**Field binding**
<img width="495" alt="Screenshot 2023-07-20 at 08 45 09" src="https://github.com/Budibase/budibase/assets/126772285/7b69cd1a-0a1d-41a5-8bcd-693ec86fcc11">

**Frontend binding on button click**
<img width="891" alt="Screenshot 2023-07-20 at 08 45 22" src="https://github.com/Budibase/budibase/assets/126772285/bfed4eab-315d-4300-9c15-86807e362c0b">

**Working version after publishing the changes**
<img width="1362" alt="Screenshot 2023-07-20 at 08 45 50" src="https://github.com/Budibase/budibase/assets/126772285/6f2cab32-7b81-4009-b9d6-e59258a475ed">

Now with the above being said, I'm going to ask the one of the engineers if this is intended behaviour or if you should be able to trigger this automation without including a field.

At the very least I think we should add a toast notification that tells you that you need to include a field with a bindable value. Instead of what it currently does which is outputting nothing/giving no feedback to the user.

I hope this makes sense and is helpful. I'll link an issue after speaking to one of the members of the engineering team if they agree with me.

**Reply by ducnhoxlatui** [reply]
Thanks man, It worked. But maybe that i think for logical, some automate can be run without filling any fields. So to say, i hope that in Budibase can plan to allow this automation without including a field.

**Reply by ConorWebb96** [reply]
No worries, yeah that was my assumption as well. My suspicious is that because theres no error notification that this might be a bug, so I will get confirmation it is then write up a issue after confirmation. 🙌 

Issue created can be found here: https://github.com/Budibase/budibase/issues/11287
